test/usecases/function_table/test_function_table.py

```diff
@@ -4,43 +4,80 @@
 import scipy
 
 
-def test_constant_1d():
```

Comment:
Could the `test_*` methods be put back? I would prefer if anything validating NMODL functionality/outputs be called `test_*`.

Reply:
Only in the form `test_everything` (which can be done); or through really annoying repetition.